src/test/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsRestClientTest.java

```diff
@@ -208,4 +211,92 @@ public TestListRequest withStart(int start) {
             return target;
         }
     }
+
+    @Test
+    public void testCreateProject() throws Exception {
+        final String name = "MyProject";
+        createTestCase(name).expectUrl("/projects/" + name).execute().verify();
+    }
+    private static ProjectCreateTestCase createTestCase(String name) {
+        return new ProjectCreateTestCase(name);
+    }
+
+    private static final class ProjectCreateTestCase {
+        private final String name;
```

Comment:
Now that I'm testing the string-based project creation, it is being used. But yes, this comment was valid at the time it was made.